src/components/organisms/Settings/WorkspaceList/index.tsx

```diff
@@ -29,7 +29,7 @@ const WorkspaceList: React.FC<Props> = ({ teamId }) => {
 
   return (
     <SettingPage teamId={teamId}>
-      <SettingsHeader title={intl.formatMessage({ defaultMessage: "Workspaces list" })} />
+      <SettingsHeader title={intl.formatMessage({ defaultMessage: "My Workspaces list" })} />
```

Comment:
My Workspace List
Above we have "My Account". Either both should have only the first word capitalized or all capitalized. Stylistically it is up to you

Reply:
yesyes